scripts/setup_hooks.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+git config core.hooksPath "./hooks"
```

Comment:
OK. What instructions does a dev need to follow in order to set up the hook? Would it make sense to include this in the main setup script so they do this automatically?